metrics/storage/README.md

```diff
@@ -0,0 +1,11 @@
+# Storage metrics tests
+
+This section includes metrics tests that measure storage bandwith under different
+conditions/configuration. These tests can be executed with different I/O operations
+such as: read, write, random read, etc.
+However, in order to find the hot spot, these tests should be used in combination
+with more tools.
+
+- `fio_job.sh`: measures the I/O bandwidth using a specific storage operation (read, write,
+   random-read, etc), and this is under a default configuration, the operations and
+   configuration can be overwritten.
```

Comment:
Let's break this up:

random-read, etc) under a default configuration. The default operations
and configuration can be overwritten.

Reply:
@MarioCarrilloA still need to fix feedback here?